### PR TITLE
Add Yandex translator support

### PR DESCRIPTION
**Pull request by bocharsky-bw**
I see Translator already supports Yandex: https://github.com/php-translation/translator/pull/7

### CODE REVIEW

File: DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -182,7 +182,7 @@ private function addAutoTranslateNode(ArrayNodeDefinition $root)
             ->arrayNode('fallback_translation')
                 ->canBeEnabled()
                 ->children()
-                    ->enumNode('service')->values(['google', 'bing'])->defaultValue('google')->end()
+                    ->enumNode('service')->values(['google', 'yandex', 'bing'])->defaultValue('google')->end()
```
Comment (bocharsky-bw) [inline DependencyInjection/Configuration.php]: Actually, we do not have `bind` support if I'm not wrong, so maybe remove it from this list for now until https://github.com/php-translation/translator/issues/1 will be implemented?

Comment (Nyholm) [reply DependencyInjection/Configuration.php]: You are correct. I just wanted have more than one value when I first write this enum (maybe it was a requirement?).

Anyhow. We should remove `bing` and use `yandex` and `google` only.

**Comment by Nyholm** [comment]
You have done a great job on this org!

Really appreciated!

**Comment by bocharsky-bw** [comment]
Glad to help ;)

**Pull request merged**